armi/reactor/grids.py

```diff
@@ -455,7 +474,11 @@ def pop(self, location: IndexLocation):
 
     @property
     def indices(self):
-        raise NotImplementedError
+        """
+        This should be at least symmetric with the way indices passed to __getitem__ on
+        Grid behaves.
```

Comment:
minor: should be in a Notes section

Reply:
Good catch; i had meant when i started staging this stuff yesterday to expound on this a little, but forgot. Will improve